src/js/app.js

```diff
@@ -69,6 +73,20 @@ const app = new Vue({
       }, 1);
     },
 
+    toggleDropdown: function () {
+      this.isDropdownOpen = !this.isDropdownOpen;
+    },
+
+    selectSort: function (order) {
+      this.isDropdownOpen = !this.isDropdownOpen;
+      this.sortCandidates(order);
+    },
+
+    includeSimilar: function() {
+      this.isIncludeSimilar = !this.isIncludeSimilar;
```

Comment:
Actually, come to think of it... We don't need to do this manually, because of `v-model="include_similar"`. It already keeps the state in the `include_similar` variable.